lms/templates/courseware/tab-view.html

```diff
@@ -35,3 +35,5 @@
 <%block name="content">
     ${HTML(fragment.body_html())}
 </%block>
+
+<%include file="sneak_peek_tos_privacy.html" args="user=user, course=course" />
```

Comment:
Screenshots before/after please.

Also, did you test this with sneak peek _disabled_?

Reply:
Before:
![screen shot 2018-05-16 at 1 23 46 pm](https://user-images.githubusercontent.com/3364609/40142054-76e4a528-590c-11e8-92f7-55163be81190.png)

After:
![screen shot 2018-05-16 at 1 23 26 pm](https://user-images.githubusercontent.com/3364609/40142064-7d19a3f8-590c-11e8-98fc-9af686711322.png)

Yes and it works fine because the conditional is in that included file...
Also, you can't reach any of the course pages if sneak peek is disabled...